src/main/java/com/abcdedu_backend/board/BoardCreateRequest.java

```diff
@@ -0,0 +1,9 @@
+package com.abcdedu_backend.board;
+
+import lombok.Getter;
+
+
+@Getter
+public class BoardCreateRequest {
+    private String name;
+}
```

Comment:
추가로 저는 request, response는 record로 사용했습니다! 모든 필드가 private final로 불변이어서 dto사용할 때 적합한 것 같더라고요. dto에는 사용하는게 좋을 것 같습니다.
```suggestion
@Getter
public record BoardCreateRequest (
    String name;
){
}
```

Reply:
오 record에 관한 이야길 덕분에 알았네요 고쳐놓겠습니다!